spec/model_spec.js

```diff
@@ -2725,6 +2755,40 @@ describe('Model', function () {
         var m = new ValidatedFoo({name: 'Foo'});
         expect(m.validateAttr('name')).toBe(false);
       });
+
+
+      describe('given a context', function() {
+        it('runs only validators for the given property name that include the context', function() {
+          var m = new ValidatedFoo({name: 'FOO', num: 34.3});
+          m.validateAttr('name', 'nameContext');
+          expect(m.ownErrors.name).toEqual(['must be lower case', 'must be greater than 5 characters']);
+        });
+
+        it('runs validators for same attribute that do not have a context', function() {
+          var m = new ValidatedFoo({name: 'FOO'});
+          m.validateAttr('name', 'nameContext');
+          expect(m.ownErrors.name).toEqual([ 'must be lower case', 'must be greater than 5 characters']);
+        });
```

Comment:
This spec is basically identical to the previous one. Change the first such that it only checks for the contextual validation error and the second to only check for the non-contextual one.